ipn/ipnlocal/local.go

```diff
@@ -3925,16 +3925,25 @@ func (b *LocalBackend) HandleQuad100Port80Conn(c net.Conn) {
 	s.Serve(netutil.NewOneConnListener(c, nil))
 }
 
+func validQuad100Host(h string) bool {
+	switch h {
+	case "",
+		tsaddr.TailscaleServiceIPString,
+		tsaddr.TailscaleServiceIPv6String,
+		"[" + tsaddr.TailscaleServiceIPv6String + "]":
```

Comment:
huh, didn't think that was a valid representation. I need to go read the RFCs again someday.

Reply:
well, it's the URL representation for IPv6 literals with colons to not look like a port number.

I didn't think browsers sent it when the implicit port number was used, but I guess they do. Without a port number.